eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -78,9 +119,12 @@ foreach ($res in $responses)
       exit 1
     }
   } 
+  
   try {
-    Remove-GitHubSourceReferences -RepoId $RepoId -Ref $branch -AuthToken $AuthToken
-    LogDebug "The branch [ $branchName ] in [ $RepoId ] has been deleted."
+    if (!$hasCentralPROpened -and $PSCmdlet.ShouldProcess("[ $branchName ] in [ $RepoId ]", "Deleting branches on cleanup script")) {
```

Comment:
Does the branch object have the sha? If so we should add that information to the log output.